src/gmail.js

```diff
@@ -3955,14 +3955,14 @@ var Gmail = function(localJQuery) {
            options.flat  boolean if true will just return an array of all recipients instead of splitting out into to, cc, and bcc
         */
         recipients: function(options) {
-            if( typeof options !== "object" ) options = {};
+            if (typeof options !== "object") options = {};
             const peopleKit = api.check.is_peoplekit_compose(this.$el);
 
             const type_selector = options.type ? "[name=" + options.type + "]" : "";
 
             const found = peopleKit ?
-                this.$el.find("tr.bzf " + type_selector + " div[data-hovercard-id]").map((_, el) => ({
-                    type: el.closest("div[name]").getAttribute("name"),
+                this.$el.find("tr.bzf " + (type_selector || "div[name]") + " div[data-hovercard-id]").map((_, el) => ({
```

Comment:
Wouldn't it be better to make `"div[name]"` the new default for `type_selector`  (line 3961) when no type is provided? Because that's effectively what this change does, right?

It would make this code a little bit easier/cleaner to read.

Reply:
I would agree if there would be no `is_peoplekit_compose` check. In other case, there is `<input name="to">`, and this selector is appended as a part of input selector (not as a parent).

Reply:
Right. I see. Let’s keep this as is too then.